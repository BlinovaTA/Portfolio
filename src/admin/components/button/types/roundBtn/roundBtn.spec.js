import roundBtn from "./roundBtn.vue";
import { mount } from "@vue/test-utils";

it("round button snapshot", async () => {
  const mountWrapper = mount(roundBtn);
  expect(mountWrapper.element).toMatchSnapshot();
});