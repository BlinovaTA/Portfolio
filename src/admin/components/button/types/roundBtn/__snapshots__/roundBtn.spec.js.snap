// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`round button snapshot 1`] = `
<button
  class="round-button-component"
  data-text="+"
  type="submit"
/>
`;
